### PR TITLE
Fast follow to address issues reported in 0.1.56

### PR DESCRIPTION
**Pull request by chrisdostert**
This PR is a fast follow to address several issues reported after releasing  0.1.56, namely:

- Fix 'opctl node kill' before you've ever run an op'
- Fix: Rate limit errors surfacing as "image not found" errors
- Don't log "not found in graph" to logs (noisy without value)
- Fixed file initialization results in incorrect (empty) file data

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -274,7 +286,7 @@ accordance with
 
 - [Named calls and needs](https://github.com/opctl/opctl/issues/643)
 
-## 0.1.29 - 2020-04-02
+## [0.1.29] - 2020-04-02
```
Comment (chrisdostert) [inline CHANGELOG.md:289]: this wasn't valid but wasn't being caught until the lint fix above

File: .markdownlint/customRule.js [hunk 2]
```diff
@@ -15,11 +15,10 @@ module.exports = {
           "lineNumber": heading.lineNumber,
           "detail": "First heading should be '# Changelog'.",
         });
-      } else if (heading.tag === "h2" && !/^## \[(\d\.\d{1,9}\.\d{1,3}(-[.A-Za-z0-9]+)?)|Unreleased\]| - \d{4}-\d{2}-\d{2}$/.test(heading.line)) {
-        // every second heading should be a version number, then a dash and a space, and then a date
+      } else if (heading.tag === "h2" && !/^## (\[\d\.\d{1,9}\.\d{1,3}(-[.A-Za-z0-9]+)?\] - \d{4}-\d{2}-\d{2})|(\[Unreleased\])$/.test(heading.line)) {
```
Comment (chrisdostert) [inline .markdownlint/customRule.js:18]: unrelated but this fixes the linter to support `[Unreleased]`, which is valid per keep a changelog spec

File: sdks/go/internal/unsudo/createFile.go [hunk 3]
```diff
@@ -14,7 +14,7 @@ func CreateFile(
 		return err
 	}
 
-	if err := os.WriteFile(fsPath, []byte{}, 0700); err != nil {
+	if err := os.WriteFile(fsPath, data, 0700); err != nil {
```
Comment (chrisdostert) [inline sdks/go/internal/unsudo/createFile.go:17]: file initialization fix

File: sdks/go/node/containerruntime/docker/isGpuSupported.go [hunk 4]
```diff
@@ -42,7 +42,7 @@ func isGpuSupported(
 
 	imageRef := "alpine"
 
-	err := pullImage(
+	if err := pullImage(
```
Comment (chrisdostert) [inline sdks/go/node/containerruntime/docker/isGpuSupported.go:45]: gpu detection image rate limiting error passthrough

File: sdks/go/node/dns/internal/resolvercfg/delete_darwin.go [hunk 5]
```diff
@@ -23,6 +23,10 @@ func Delete(
 				return err
 			}
 
+			if d == nil {
```
Comment (chrisdostert) [inline sdks/go/node/dns/internal/resolvercfg/delete_darwin.go:26]: fix for panic on `opctl node kill` if never run an op

File: website/docs/reference/opspec/op-directory/op/call/container/index.md [hunk 6]
```diff
@@ -50,15 +49,10 @@ An object for which each key is an absolute path in the container and each value
 |[file initializer](../../../../types/file.md#initialization)|Evaluate and mount|
 
 ### name
-A [string initializer](../../../../types/string.md#initialization) defining a name by which the container can be resolved on the opctl network.
+A [string initializer](../../../../types/string.md#initialization) defining a name by which the container can be reached by other opctl containers and opctl host nodes.
 
 > if multiple containers are given the same name, network requests will be distributed (load balanced) across them. 
 
-### ports
```
Comment (chrisdostert) [inline website/docs/reference/opspec/op-directory/op/call/container/index.md:57]: ports deprecated as of last release so remove from docs

**Review by zaknoori — APPROVED** [review]

**Review by zaknoori — APPROVED** [review]

**Review by zaknoori — APPROVED** [review]

**Pull request merged**